src/styles/block/stars.scss

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+
+  &__star {
+    display: flex;
+    background-image: url(../images/star.svg);
+    background-position: center;
+    height: 16px;
+    width: 16px;
+    background-repeat: no-repeat;
+    margin-left: 4px;
+  }
+
+  &__star:nth-child(-n + 4) {
```

Comment:
you should reuse logif from previous task for all possible values of modifier. It should work with 1,2,3,4,5 active stars

Reply:
I don't understand how to do it. If I apply all the stars--n blocks, I will have 5 rows of stars.